apps/sms/views/shared/js/dialog.js

```diff
@@ -107,7 +107,7 @@ var Dialog = function(params) {
     event.preventDefault();
   });
 
-  this.form.addEventListener('transitionend', function(event) {
+  window.eventSafety(this.form, 'transitionend', function(event) {
     var form = event.target;
```

Comment:
I see we  can't used `event` when we use `eventSafety` as it's not always set...